### DISCUSSION TITLE
What is the `Dialog.Positioner` used for?

### DISCUSSION THREAD

**Discussion by ivancuric** [chakra-ui/ark#2818] [Q&A] [answered]
So far I haven't found a single use for it. Is it just a wrapper `div` for the backdrop and content? If so, why not just use a `div` instead of a custom component?

**Comment by segunadebayo** [comment, marked as answer]
The positioner is indeed a UI only component. It’s mostly just a div like you mentioned. But it’s useful for positioning the dialog content, and managing overflowing content.

You won’t need it every time but it comes in handy when needed.

**Reply by ivancuric** [reply]
If it's just a `div` why have it as an undocumented framework-specific component? For semantic meaning?
